scene/resources/surface_tool.cpp

```diff
@@ -724,6 +724,26 @@ void SurfaceTool::create_from(const Ref<Mesh> &p_existing, int p_surface) {
 	material = p_existing->surface_get_material(p_surface);
 }
 
+void SurfaceTool::create_from_blend_shape(const Ref<Mesh> &p_existing, int p_surface, const String p_blend_shape_name) {
+	clear();
+	primitive = p_existing->surface_get_primitive_type(p_surface);
+	Array arr = p_existing->surface_get_blend_shape_arrays(p_surface);
+	Array blend_shape_names;
+	int32_t shape_idx = -1;
+	for (size_t i = 0; i < p_existing->get_blend_shape_count(); i++) {
```

Comment:
https://travis-ci.org/godotengine/godot/jobs/499318138

You should just use `int i`, there's no reason to use `size_t` here.

Reply:
Changed int32_t.